site/_posts/2021-03-08-establishing-reliability-as-code-via-sonobuoy.md

```diff
@@ -0,0 +1,112 @@
+# Establishing Reliability-As-Code via Sonobuoy
+
+We are happy to introduce the [Reliability Scanner for Kubernetes as a Sonobuoy Plugin](https://github.com/vmware-tanzu/sonobuoy-plugins/tree/master/reliability-scanner)! It is a light-weight Go program that includes an extensible set of reliability assessments, or checks, performed against various components of a cluster, such as Pods, Namespaces, Services, etc. The Reliability Scanner runs as a container that serves as a Sonobuoy plugin and uses a configuration file to define customized checks. Kubernetes cluster operators can then configure appropriate constraints to run reliability scanner checks against their clusters. This project is based upon efforts and practices of [VMware’s Customer Reliability Engineering (CRE) team](https://tanzu.vmware.com/content/blog/hello-world-meet-vmware-cre).
+
+This article provides a walk-through of the components usedby the initial set of checks recommended by VMware CRE.  You can create your own customized checks or extend the existing checks. 
+
+## Probes
+Probes are periodic checks against containers that run our services and notify the kubelet when the container is alive and ready to accept traffic. Probes help Kubernetes make more informed decisions about the current status of one or many particular Pods behind a Service. 
+
+There are three kinds of probes: 
+startupProbe -  confirms the application within the container is available
+livelinessProbe confirms the container is in a running state
+readinessProbe - confirms the container is ready to respond to requests 
```

Comment:
Looks like this isn't formatted properly -- https://deploy-preview-1217--sonobuoy.netlify.app/establishing-reliability-as-code-via-sonobuoy/

I'd suggest this:

```suggestion
There are three kinds of probes: 

- `startupProbe`: confirms the application within the container is available
- `livelinessProbe`: confirms the container is in a running state
- `readinessProbe`: confirms the container is ready to respond to requests 
```